src/common/commonutils/UserUtils.c

```diff
@@ -828,28 +834,28 @@ int RemoveUser(SimplifiedUser* user, bool removeHome, OsConfigLogHandle log)
     }
     else if (0 == user->userId)
     {
-        OsConfigLogInfo(log, "RemoveUser: cannot remove user with uid 0 ('%s' %u, %u)", user->username, user->userId, user->groupId);
+        OsConfigLogInfo(log, "RemoveUser: cannot remove user with uid 0 (%u, %u)", user->userId, user->groupId);
         return EPERM;
     }
 
     if (NULL != (command = FormatAllocateString(commandTemplate, removeHome ? "-f -r" : "-f", user->username)))
     {
         if (0 == (status = ExecuteCommand(NULL, command, false, false, 0, 0, NULL, NULL, log)))
         {
-            OsConfigLogInfo(log, "RemoveUser: removed user '%s' (%u, %u, '%s')", user->username, user->userId, user->groupId, user->home);
+            OsConfigLogInfo(log, "RemoveUser: removed user %u", user->userId);
```

Comment:
Should rename to "removed userId" instead of "removed user"

Reply:
User accounts are removed, not just ids.